
# PBMC68k project for MBP1413 (graduate level course in computational biology)

The task was to train machine learning classifiers from raw scRNA-seq outputs to assign cell type labels, instead of the PCA/clustering approach commonly used for analysis of RNA-seq data.

The R_script_preprocessing directory contains modified code from the original GitHub repository used to generate figures found in the Nature Communications paper (https://github.com/10XGenomics/single-cell-3prime-paper/tree/master/pbmc68k_analysis). These R scripts are modified to extract the raw RNA-seq data and other relevant information of the pure and PBMC68k population. The modified portion of the scripts can be found at the end of each file (except for util.R, which has not been modified).


The python_script_machine_learning directory contains jupyter notebooks with code used to generate and train the MLP and VAE models for the cell-type classification task. Scripts used to generate all figures in the report are also included in these notebooks.
